......................F.......F.F......F..F.F.
Time: 0.106
There were 6 failures:
1) test19(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test19(RandoopTest0.java:245)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
2) test12(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test12(RandoopTest0.java:152)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
3) test11(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test11(RandoopTest0.java:140)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
4) test5(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test5(RandoopTest0.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
5) test3(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test3(RandoopTest0.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)
6) test2(RandoopTest0)junit.framework.AssertionFailedError: null
	at RandoopTest0.test2(RandoopTest0.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at RandoopTest.main(RandoopTest.java:8)

FAILURES!!!
Tests run: 40,  Failures: 6,  Errors: 0

Cycle length is4
before
after
Reached this
# of deadlocks detected 0
THE SIZE OF THE TRACE OMEN 35044
Exiting analysis
